.env.development

```diff
@@ -1,6 +1,7 @@
 ZETKIN_APP_DOMAIN=http://www.dev.zetkin.org
 ZETKIN_GEN2_ORGANIZE_URL=http://organize.dev.zetkin.org
 ZETKIN_GEN2_CALL_URL=http://call.dev.zetkin.org
+ZETKIN_LOGIN_URL=http://login.dev.zetk.in
```

Comment:
This is not necessary. There is already a URL at `/login` which redirects to the correct login URL, so we don't need another environment variable, and this file should not be part of this PR.